analyses/chromothripsis/03-plot-chromothripsis-by-histology.Rmd

```diff
@@ -42,53 +43,59 @@ chromoth_per_sample <- read_tsv(file.path(analysis_dir, "results", "chromothrips
 histology_label_mapping <- read_tsv(
   file.path(root_dir, "figures", "palettes", "histology_label_color_table.tsv")) %>% 
   # Select just the columns we will need for plotting
-  select(Kids_First_Biospecimen_ID, display_group, display_order, hex_codes)  
-  # # Reorder display_group based on display_order --- skip for now because I'm ordering based on chromothripsis data
-  # dplyr::mutate(display_group = forcats::fct_reorder(display_group, display_order))
+  select(Kids_First_Biospecimen_ID, display_group, hex_codes, cancer_group, cancer_group_hex_codes)
 
 # Merge chromothripsis data and histology display groups
 chromoth_histology <- inner_join(chromoth_per_sample, histology_label_mapping, by="Kids_First_Biospecimen_ID")
 ```
 
 
-### Summarize chromothripsis data by histology and reorder histologies
+### Summarize chromothripsis data by histology (display_group and cancer_group)
 ```{r}
-## For each histology, calculate proportion of tumors with at least one chromothripsis region (`prop`)
-# Will be used in first plot and to order the histologies in all plots
+## For each histology (display_group), calculate proportion of tumors with at least 
+## one chromothripsis region (`prop`)
+## This will be used in first plot, and to reorder display_group in subsequent plots
 chromoth_histology_summary <- chromoth_histology %>%
   count(any_regions_logical, display_group, hex_codes) %>%
   tidyr::spread(key = any_regions_logical, value = n, fill=0) %>%
-  group_by(display_group, hex_codes) %>%
-  mutate(group_size = sum(`TRUE`, `FALSE`)) %>%
-  mutate(prop = `TRUE` / group_size) %>%
-  mutate(labels = paste0(`TRUE`, " / ", group_size)) 
```

Comment:
This section did not actually change, I just updated the variable names to contrast with the code I added.